modules/openapi-generator/src/main/resources/typescript-axios/package.mustache

```diff
@@ -24,7 +24,7 @@
   },
   "devDependencies": {
     "@types/node": "^8.0.9",
-    "typescript": "^2.4" 
+    "typescript": "^3.2"
```

Comment:
which feature requires typescript 3.2? I would widen the restriction as much as possible

Reply:
Reverted at 9818ee80b249d388b61211f5fccddf8dad2c8eea -
Thought the spread operators was a later language feature